README.md

```diff
@@ -125,3 +125,7 @@ Here is a list of the more detailed features including a link to the documentati
  * scales from small indoor-sized to world-wide-sized graphs
  * Find nearest point on street e.g. to get elevation or 'snapp to road'
  * Enables the [map matching component](https://github.com/graphhopper/map-matching) (flexibility only)
+
+# Attribution
```

Comment:
I think all of the GTFS feeds in Germany require attribution in one way or another so we can't avoid attribution statements in the long run.
I've added a separate `README.md` for `reader-gtfs`, moved the attribution statement there.